buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -54,6 +54,12 @@ object Versions {
   const val gradleCurrentVersion: String = "5.6.1"
 }
 
+object Plugins {
```

Comment:
if this was added by hand, i'd advise putting this in a separate file cuz it'll get wiped as soon as dependencies are updated / you use buildSrcVersions to find newer versions of dependencies

Reply:
ah ok thanks.  Yes it was added manually.  Not to familiar with buildSrcVersions